packages/polaris-core/src/plugins/extensions/extensions-listener.ts

```diff
@@ -27,14 +27,19 @@ export class ExtensionsListener implements GraphQLRequestListener<PolarisGraphQL
             if (!this.shouldAddWarningsToExtensions) {
                 context.returnedExtensions.warnings = undefined;
             }
+            if (context.errors) {
+                const errors: string[] = [];
+                context.errors?.forEach((err) => errors.push(err.toString()));
+                context.returnedExtensions.errors = errors;
```

Comment:
why did you used this complex cast into array?

Reply:
cause errors on the context object are of type polarisError, and I want to convert it to string array, so I could push it to the metadata buffer